product_barcode_generator/product_view.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        
+        <record id="view_product_category_simple_ean_form" model="ir.ui.view">
+            <field name="name">product.category.form</field>
+            <field name="model">product.category</field>
+            <field name="inherit_id" ref="product.product_category_form_view"/>
+            <field name="arch" type="xml">
+                <field name="type" position="after">
+                    <field name="ean_sequence_id" domain="[('barcode_sequence', '=', 1)]"
+                        context="{'default_name': name and 'EAN ' + name or 'EAN', 'default_barcode_sequence': 1}"/>
+                    <newline/>
+                </field>
+            </field>
+        </record>
+        
+		<record id="view_product_simple_ean_form" model="ir.ui.view">
+			<field name="name">product.product.form</field>
+			<field name="model">product.product</field>
+			<field name="inherit_id" ref="product.product_normal_form_view"/>
+			<field name="arch" type="xml">
+				<field name="ean13" position="after">
+                    <field name="ean_sequence_id" domain="[('barcode_sequence', '=', 1)]"
+                        context="{'default_name': name and 'EAN ' + name or 'EAN', 'default_barcode_sequence': 1}"
+                        class="oe_edit_only"/>
```

Comment:
Why oe_edit_only? it's strange having to enter in edit mode to see that the product is linked to a specific sequence.